packages/application-shell/src/components/quick-access/butler/butler.js

```diff
@@ -455,7 +455,12 @@ export class Butler extends React.Component {
                 </div>
               );
 
-            if (!this.state.results) return null;
+            if (this.state.results.length === 0 && this.state.searchText !== '')
```

Comment:
Would make sense to trim the value to check if it's empy? 🤔

Reply:
Fair point not sure. It's not done anywhere else. So I presume it would be a separate fix.